content/millennium/dstu2/diagnostic/observation.md

```diff
@@ -69,7 +70,9 @@ Notes:
   - May be a list of comma separated values. A system must be provided for each code.
   - Searches Observation.code and Observation.component.code.
 
-- The `date` parameter may be provided up to two times, and must use the `eq`, `ge`, `gt`, `le`, or `lt` prefixes. When a value is provided without a prefix, an implied `eq` prefix is used. When provided twice, the lower value must have a `ge` or `gt` prefix and the higher value must have an `le` or `lt` prefix.
+- The `date` and `_lastUpdated` parameters may be provided up to two times, and must use the `eq`, `ge`, `gt`, `le`, or `lt` prefixes. When a value is provided without a prefix, an implied `eq` prefix is used. When provided twice, the lower value must have a `ge` or `gt` prefix and the higher value must have an `le` or `lt` prefix.
+
+- The `date` and `_lastUpdated` parameters may not be provided together.
```

Comment:
can _lastUpdated be provided with all the other parameters except for date? That's how I would interpret the doc, but want to confirm that's true :)